app/src/sharedTest/java/org/oppia/android/app/topic/TopicFragmentTest.kt

```diff
@@ -343,7 +345,7 @@ class TopicFragmentTest {
   }
 
   @Test
-  fun testTopicFragment_configurationChange_showsDefaultTabAndItsContent() {
+  fun testTopicFragment_configChange_showsDefaultTabAndItsContent() {
```

Comment:
Thanks for fixing this `configChange`.
Could you please check if there is an issue filled to fix this, if yes, include that issue in this PR.

Reply:
There is no issue open related to topicFragmentTest.

Reply:
I found no issue to fix this.